### ISSUE TITLE
Why are the values of `padding_mask` all 1 in UEA heartbeat `classification` task?

### ISSUE THREAD

**Issue by rdyan0053** [thuml/Time-Series-Library#149]
When I debug the code
```python
for i, (batch_x, label, padding_mask) in enumerate(train_loader):
    iter_count += 1
    model_optim.zero_grad()

    batch_x = batch_x.float().to(self.device)
    padding_mask = padding_mask.float().to(self.device)
    label = label.to(self.device)
```
the values of `padding_mask` all are 1. Why?

Another question: How `padding_mask` is generated？
from the code`for i, (batch_x, label, padding_mask) in enumerate(train_loader):`, we can know `padding_mask` from the `train_set`.
But I check the generating `data_set` code and find the class `UEAloader`. Its `__getitem__` code:
```python
    def __getitem__(self, ind):
        return self.instance_norm(torch.from_numpy(self.feature_df.loc[self.all_IDs[ind]].values)), \
               torch.from_numpy(self.labels_df.loc[self.all_IDs[ind]].values)
```
Here return two variables: `data` and `label`. 

But in `for i, (batch_x, label, padding_mask) in enumerate(train_loader):`, there are three variables: `batch_x`, `label` and `paddding_mask`. So I don't understand how the `padding_mask` is generated?

**Comment by rdyan0053** [comment]
well, I know the answer of the second question.
```python
data_loader = DataLoader(
    data_set,
    batch_size=batch_size,
    shuffle=shuffle_flag,
    num_workers=args.num_workers,
    drop_last=drop_last,
    collate_fn=lambda x: collate_fn(x, max_len=args.seq_len)
)
```
in the `data_factory.py`, the `collate_fn` creates the `padding_masks`.To be precise, use the `padding_mask` function to create variable `padding_masks`.

the code comments:
```python
"""
    Used to mask padded positions: creates a (batch_size, max_len) boolean mask from a tensor of sequence lengths,
    where 1 means keep element at this position (time step)
"""
```
I guess I can understand it
